src/commands/antiflood.ts

```diff
@@ -32,7 +28,7 @@ registerCommand({
       return;
     }
 
-    let elapsed = now - bucket.lastTS;
+    const elapsed: number = now - bucket.lastTS;
```

Comment:
Again, why the explicit type here D:

Reply:
I mean explicit *return* types make sense, but explicit types when they do not add value are only going to be a pain while refactoring. When there are more complex types in the mix, or you explicitly want to enforce some type, sure.